doc/api/cli.md

```diff
@@ -586,6 +586,15 @@ added: v12.0.0
 Set default [`tls.DEFAULT_MIN_VERSION`][] to 'TLSv1.1'. Use for compatibility
 with old TLS clients or servers.
 
+### `--tls-min-v1.2`
+<!-- YAML
+added: REPLACEME
```

Comment:
Will the `REPLACEME` value be the wrong version given that this has already landed in earlier versions? Should we put the actual `11.x` version here now?

Reply:
REPLACEME values are only accurate on the release branch they land in (last time I checked).